minerals/startarmcord

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+FLAGS='--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --enable-features=UseSkiaRenderer --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations'
```

Comment:
I suggest to omit the Wayland backend. It's still broken for many users.

Reply:
That might be better done in the permissions themselves, as `--ozone-platform-hint=auto` decides based on what's available. If the arguments are kept here, users can enable the Wayland backend simply by changing the Flatpak permissions.

Reply:
Oh, that's much better.

Reply:
Interesting, I've changed the appropriate permissions and it refuses to use X.

Reply:
Maybe it goes off of the XDG_SESSION_TYPE variable? That could be troublesome.

Reply:
I'm not sure, I never got that problem on other Discord clients.

Reply:
I'll just nuke everything Wayland-related, and add a README later on addressing this.